config/configtelemetry/configtelemetry.go

```diff
@@ -36,6 +36,8 @@ const (
 	levelDetailedStr = "detailed"
 )
 
+// Deprecated: UseOpenTelemetryForInternalMetrics has been deprecated. Uses feature flag in
+// service.useOtelForInternalMetricsfeatureGateID to handle this feature
```

Comment:
the "feature flag" is `telemetry.useOtelForInternalMetrics` not the internal variable.

Reply:
updated